### ISSUE TITLE
Testnet v3.3: Unhandled top-level exception: $exn / Unconsumed item in cache: $cache

### ISSUE THREAD

**Issue by bakebrain** [MinaProtocol/mina#6049]
## Summary

- started coda daemon via `coda daemon -peer $SEED1`
- set staking: `coda client set-staking -public-key $CODA_PUBLIC_KEY`
- `SEED1` and `CODA_PUBLIC_KEY` are set.

### Error

```
2020-09-19 15:33:43 UTC [Info] validating a new pubsub message ...
2020-09-19 15:33:43 UTC [Info] validating a new pubsub message ...
2020-09-19 15:33:43 UTC [Info] validating a new pubsub message ...
2020-09-19 15:33:43 UTC [Info] validating a new pubsub message ...
2020-09-19 15:33:43 UTC [Info] validating a new pubsub message ...
2020-09-19 15:33:46 UTC [Info] why u fail to validate :(
2020-09-19 15:33:49 UTC [Warn] writing to closed pipe "(\"valid transitions\")"
2020-09-19 15:33:49 UTC [Fatal] Unhandled top-level exception: $exn
Generating crash report
        exn: "(monitor.ml.Error\n  (\"write to closed pipe\"\n    (pipe\n      ((id 98) (buffer ()) (size_budget 0) (pushback (Full ()))\n        (num_values_read 0) (blocked_flush
es ()) (blocked_reads ())\n        (closed (Full ())) (read_closed Empty) (consumers ())\n        (upstream_flusheds ()))))\n  (\"Raised at file \\\"src/error.ml\\\" (inlined), lin
e 9, characters 14-30\"\n    \"Called from file \\\"src/error.ml\\\", line 11, characters 19-40\"\n    \"Called from file \\\"src/pipe.ml\\\", line 510, characters 2-15\"\n    \"Ca
lled from file \\\"src/deferred1.ml\\\", line 17, characters 40-45\"\n    \"Called from file \\\"src/job_queue.ml\\\" (inlined), line 131, characters 2-5\"\n    \"Called from file 
\\\"src/job_queue.ml\\\", line 171, characters 6-47\"\n    \"Caught by monitor coda\"))"
2020-09-19 15:33:49 UTC [Info] Starting Bootstrap Controller phase
2020-09-19 15:33:49 UTC [Error] Unconsumed item in cache: $cache
        cache: "cached item was not consumed (cache name = \"Transition_handler__Unprocessed_transition_cache\")"
  ☠  Coda Daemon crashed.
   The Coda Protocol developers would like to know why!
    Please:
      Open an issue:
        <https://github.com/CodaProtocol/coda/issues/new>
      Briefly describe what you were doing and attach the crash report /home/bakebrain/.coda-config/coda_crash_report_2020-09-19_15-33-49.843948.tar.gz
```
   
### version
```
bakebrain@coda-daemon-1:/usr/local$ coda -version
Commit 4395e91 on branch master
```

### status just before crash
```
Global number of accounts:       1017
Block height:                    1
Max observed block length:       1
Local uptime:                    5m58s
Ledger Merkle root:              jxH4Qmdt5jFUoVupEihNvVBpSh2oiN4cSjDqgpS7oq9Bsdat4U2
Protocol state hash:             3NLwBcsoZBdUGtndj14Cts5SRDwx3xvk4jEYAqBNRJ87BVzKuesM
Chain id:                        b90b8b744074d8cc9d864925b58d1f882b1c2e89a46245be3e0a969d39de43a0
Git SHA-1:                       4395e9162b55323e76141c174ed585a0bf8cce81
Configuration directory:         /home/bakebrain/.coda-config
Peers:                           5 (104.196.56.76:10008 34.75.134.50:10007 34.74.99.242:10001 104.196.122.249:10009 34.74.157.233:10005)
User_commands sent:              0
SNARK worker:                    None
SNARK work fee:                  25000000
Sync status:                     Synced
Block producers running:         1 (B62qjx6vxaWNfn75FB295M6K56brKA6eq3cHHibd8EJz5Wkcw2PWxrG)
Best tip consensus time:         epoch=0, slot=0
Next block will be produced in:  None this epoch… checking at in 4.452h
Consensus time now:              epoch=0, slot=390
Consensus mechanism:             proof_of_stake
Consensus configuration:         
        Delta:                     3
        k:                         20
        c:                         8
        c * k:                     160
        Slots per epoch:           480
        Slot duration:             3m
        Epoch duration:            1d
        Genesis state timestamp:   2020-09-18 20:00:20.000000Z
        Acceptable network delay:  9m
Addresses and ports:             
        External IP:    35.242.208.139
        Bind IP:        0.0.0.0
        Libp2p PeerID:  12D3KooWQKhxjckkDVBBrF5ncHTTkz56UBxQpfhL4XrUt91CMZHk
        Libp2p port:    8302
        Client port:    8301
```

#### log
[coda_crash_report_2020-09-19_15-33-49.843948.tar.gz](https://github.com/CodaProtocol/coda/files/5250118/coda_crash_report_2020-09-19_15-33-49.843948.tar.gz)

#### discord
powersource#7803

**Comment by garethtdavies** [comment]
Also experiencing this and is also mentioned in this issue as #4 https://github.com/CodaProtocol/coda/issues/4480

**Comment by mrmr1993** [comment]
Closing, this was fixed in #6357.
